package.json

```diff
@@ -26,7 +26,7 @@
     "@edx/brand": "npm:@edx/brand-openedx@1.1.0",
     "@edx/frontend-component-footer-edx": "3.0.10",
     "@edx/frontend-platform": "1.9.0",
-    "@edx/paragon": "11.1.0",
+    "@edx/paragon": "^14.4.0",
```

Comment:
Since some components are deprecated in the latest version of paragon, let's not update it to the latest version right now and instead use `13.17.5`. Also, remove the caret sign.

With this version, you might not need to update the test snapshots as well.

Reply:
Also, this version is not yet supported by `"@edx/frontend-platform": "1.9.0"`

Reply:
thanks @waheedahmed for the context